airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -1862,6 +1862,13 @@ class BigQueryUpsertTableOperator(BaseOperator):
     :type delegate_to: str
     :param location: The location used for the operation.
     :type location: str
+    :param encryption_configuration: [Optional] Custom encryption configuration (e.g., Cloud KMS keys).
```

Comment:
Nit, use `(Optional)` not `[Optional]` to keep docs consistent.

Reply:
Thanks for your comments! Yes, it makes sense to me. Does this mean that we need to close this PR since we don’t need to modify Airflow at all, or does it mean that our solution here probably wouldn’t fix the CMEK supporting issue in upsert function so that we need to keep digging into it?